pyiron_base/project/archiving/export_archive.py

```diff
@@ -38,7 +38,11 @@ def update_project(project_instance, directory_to_transfer, archive_directory, d
 
 
 def copy_files_to_archive(
-    directory_to_transfer, archive_directory, compress=True, copy_all_files=False, arcname=None
+    directory_to_transfer,
+    archive_directory,
+    compress=True,
+    copy_all_files=False,
+    arcname=None,
```

Comment:
It would be great to add type hints - now that we use co-pilot this should be easy